SharedSources/MediaLibraryModel/MediaLibraryBaseModel.swift

```diff
@@ -25,7 +25,7 @@ protocol MediaLibraryBaseModel {
     func sort(by criteria: VLCMLSortingCriteria)
 }
 
-protocol MLBaseModel: MediaLibraryBaseModel {
+protocol MLBaseModel: class, MediaLibraryBaseModel {
```

Comment:
never seen this, what does this do ?

Reply:
It makes it a class only protocol, it's the same thing as AnyObject but actually I will change it to AnyObject. 😄

Reply:
yeah I actually read up on it